Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj

```diff
@@ -23,7 +23,7 @@
 
 	<ItemGroup Condition="'$(TargetFrameworkIdentifier)' == 'MonoAndroid' or '$(TargetFrameworkIdentifier)' == 'Xamarin.iOS' or '$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(TargetFrameworkIdentifier)' == 'Xamarin.Mac'">
 		<PackageReference Include="Uno.UI"
-											Version="3.4.0" />
+											Version="3.8.11" />
```

Comment:
Don't update Uno if not absolutely necessary. It'll force everyone using WCT to upgrade to latest.